.github/workflows/basic.yml

```diff
@@ -35,23 +40,56 @@ jobs:
         sudo apt-get -y install zlib1g-dev libssl-dev uuid-dev libcurl4-openssl-dev
         mkdir $CMAKE_BUILD_DIR
 
+    - name: Install Prereqs - MacOS and OpenSSL Version 1.1
```

Comment:
Can you merge this with the `Install Prereqs - MacOS` task? You should be able to use `${{matrix.openssl-version}}` as the environment variable in bash.